packages/itwin/grouping-mapping-widget/README.md

```diff
@@ -29,3 +29,37 @@ import { GroupingMappingProvider } from "@itwin/grouping-mapping-widget";
   uiProviders={[new GroupingMappingProvider()]}
 />
 ```
+
+## UI Configuration
+
+This package provides an interface to define your own group creation method. You can use your own UI component to create or update the query.
+
+An example of group custom UI provider, [ManualUIComponent](./src/widget/components/provider/ManualUIComponent.tsx) helps user to manually define ECSQL query for groups.

```

Comment:
might want to make this an absolute path to the file on gh

Reply:
why do we want absolute path? It won't work in cospace, is it?

Reply:
this relative path is only relevant when you have the repo open
when this is ublished to npm or while looking at this when the pkg is isntalled in node_modules, this path wont work